node/src/Commands.ts

```diff
@@ -1584,6 +1584,56 @@ export function createBLPop(
     return createCommand(RequestType.BLPop, args);
 }
 
+/**
+ * @internal
+ */
+export function createFCall(
+    func: string,
+    keys?: string[],
+    args?: string[],
+): command_request.Command {
+    const params: string[] = [];
+    params.push(func);
+
+    if (keys !== undefined) {
+        params.push(keys.length.toString());
+        params.push(...keys);
```

Comment:
nit: use `concat` instead of `...` to [avoid issues](https://www.educative.io/answers/spread-operator-vs-arrayprototypeconcat-in-javascript)